soccer/Robot.cpp

```diff
@@ -216,6 +216,14 @@ void OurRobot::lineKick(Point target) {
         std::make_unique<Planning::LineKickCommand>(std::move(target));
 }
 
+void OurRobot::intercept(Point target) {
+    if (!visible) return;
+
+    disableAvoidBall();
+    _motionCommand =
+        std::make_unique<Planning::InterceptCommand>(std::move(target));
```

Comment:
Don't move point. It's faster to just copy it actually than to reference pass it. Also it has nothing thats movable about it.

Reply:
If you could fix the one in the LineKickplanenr too. That would be nice.